testdata/project-v3-config/main.go

```diff
@@ -77,6 +77,14 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err = (&controllers.CaptainReconciler{
+		Client: mgr.GetClient(),
+		Log:    ctrl.Log.WithName("controllers").WithName("Captain"),
+		Scheme: mgr.GetScheme(),
+	}).SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "Captain")
+		os.Exit(1)
+	}
```

Comment:
/hold

I should not duplicate it since the --force will re-create the file.

Reply:
Thread: https://github.com/kubernetes-sigs/kubebuilder/pull/1903#discussion_r542919409

/hold cancel